### DISCUSSION TITLE
Add "Status" dropdown to actual invoice screen for easy resolutions

### DISCUSSION THREAD

**Discussion by astupidmoose** [btcpayserver/btcpayserver#2917] [General]
Hello all, 

We get a lot of "Unusual" invoices and are constantly in this screen. One thing we use a lot is the "Status" dropdown menu to either mark items complete or invalid. 

The issue for us is when an invoice is marked as unusual, we usually go into the details to see exactly what has happened (When was the invoice paid, what was underpaid, what was the rate etc..). From here we make a determination of if we want process the order, or mark it invalid, however in order to do that, we then need to go back out to the invoice list and do so. 

It would be way easier if the dropdown was simply added to the invoice screen. I've added some images to visualize, as well as a couple spots it might make sense to integrate this onto this screen. 

![image](https://user-images.githubusercontent.com/3916318/134564535-834aec92-4009-497a-8bbf-8cc5cd1a2ea7.png)

![image](https://user-images.githubusercontent.com/3916318/134565225-0ce61b34-64aa-4d6e-9a61-2509ec6a832f.png)

**Comment by pavlenex** [comment]
Additionally add a tooltip explaining the status in a human friendly way so that merchants know how to react.

**Comment by bolatovumar** [comment]
PR: https://github.com/btcpayserver/btcpayserver/pull/2923
